test/libsolidity/syntaxTests/userDefinedValueType/wrap_unwrap_via_contract_name_different.sol

```diff
@@ -0,0 +1,8 @@
+contract C { type T is uint; }
+library L { type T is uint; }
+contract D
+{
+    C.T x = L.T.wrap(uint(1));
+}
+// ----
+// TypeError 7407: (86-103): Type user defined type T is not implicitly convertible to expected type user defined type T.
```

Comment:
Yeah, ok, this works, but the error message sucks :-D.

Reply:
How would you change it?

Reply:
Maybe should also include `L.T` and `C.T` in the `Type::toString()`.

Reply:
If I were to change it, I'd do the same as we currently do for enums... but @chriseth and I just said, we don't really care, people should do this on file-level with nice names anyways :-).

Reply:
Here's the PR for this: https://github.com/ethereum/solidity/pull/11957